README.md

```diff
@@ -1 +1,70 @@
-# Hello-world-project
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello World
+
+> Description the project.
+
+
+## Built With
+
+- Major languages
+- Frameworks
+- Technologies used
+
+## Live Demo (if available)
```

Comment:
- [ ] I kindly suggest that you remove unused headers so as to make your project look neat.